### ISSUE TITLE
No shared framework schemes found

### ISSUE THREAD

**Issue by bastilimbach** [fahlout/PasswordExtension#2]
If you try to install this framework using Carthage the following error occurs:
```bash
$ carthage update --platform iOS

Dependency "PasswordExtension" has no shared framework schemes for any of the platforms: iOS
```

**Carthage Version:** 0.28.0

**Comment by fahlout** [comment]
I have added a prebuilt version of the framework for Carthage to the project. Let me know if that works for you. That seemed to be the only way the CocoaPods template I have used for this project might cooperate with Carthage support. I hope this will work for you. Thanks for bringing up the issue.

**Comment by bastilimbach** [comment]
Thanks for the quick response. This fixed the issue but I think it is far from optimal because you need to build the framework manually every time you make a change, am I right?

I think Carthage throws this issue because you do not have your XCode project in the root directory of this repo. Maybe this will help: [https://stackoverflow.com/questions/35054788/carthage-no-shared-framework-schemes-for-ios-platform-for-my-own-framework](https://stackoverflow.com/questions/35054788/carthage-no-shared-framework-schemes-for-ios-platform-for-my-own-framework)

**Comment by fahlout** [comment]
That's correct. I'll try to figure it out to where I won't need to do it manually every time.

**Comment by bastilimbach** [comment]
I would love to help you with that but I never published a framework using Carthage before.

Maybe it is as simple as this: [https://github.com/httpswift/swifter/issues/62#issuecomment-164775434](https://github.com/httpswift/swifter/issues/62#issuecomment-164775434)

**Comment by fahlout** [comment]
No worries. I usually use CocoaPods, so I'm not too familiar with Carthage, but wanted to give people a choice since I do believe this framework is very helpful to have in an app.
